addons/website/models/website_snippet_filter.py

```diff
@@ -41,12 +42,19 @@ def _check_field_names(self):
                 if not field_name.strip():
                     raise ValidationError(_("Empty field name in %r") % (record.field_names))
 
+    @api.model
+    def search_read(self, domain=None, fields=None, offset=0, limit=None, order=None):
```

Comment:
not needed anymore, the controller do the job for our use case, and from this way, we avoid override of orm method without plusvalue.